integration/token/fungible/support.go

```diff
@@ -1233,3 +1259,32 @@ func CheckPrometheusMetrics(ii *integration.Infrastructure, viewName string) {
 		Expect(v.Value).NotTo(Equal(model.SampleValue(0)))
 	}
 }
+
+func Conversion(network *integration.Infrastructure, wpm *WalletManagerProvider, user *token3.NodeReference, wallet string, typ token.Type, auditor *token3.NodeReference, issuer *token3.NodeReference, expectedErrorMsgs ...string) string {
```

Comment:
Maybe convert is very generic. it might create confusion and it might also clash with other possible conversions in the future (e.g. between currencies).

maybe we could call it `upgrade` or `promote`.

Reply:
went for `upgrade`